examples/plotting_with_geoplot.py

```diff
@@ -48,10 +48,17 @@
 ###############################################################################
 # ``polyplot`` is trivial and can only plot the geometries you pass to it. If
 # you want to use color as a visual variable, specify a ``choropleth``. Here
-# we sort GDP per person by country into five buckets by color.
+# we sort GDP per person by country into five buckets by color, using
+# "quantiles" binning from the `Mapclassify <https://pysal.org/mapclassify/>`_
+# library.
 
+import mapclassify
+gpd_per_person = world['gdp_md_est'] / world['pop_est']
+scheme = mapclassify.Quantiles(gpd_per_person, k=5)
+
+import mapclassify
```

Comment:
This import is duplicated. You already imported mc on line 55.

Reply:
Done.